includes/admin/customers/class-customer-addresses-table.php

```diff
@@ -66,9 +66,10 @@ public function column_default( $item, $column_name ) {
 		switch ( $column_name ) {
 
 			case 'type' :
-				$value = ( 'primary' === $item['type'] )
-					? esc_html_e( 'Primary',   'easy-digital-downloads' )
-					: esc_html_e( 'Secondary', 'easy-digital-downloads' );
+				$value = ucfirst( $item['type'] );
```

Comment:
This won't be translatable now. Perhaps we translate the values we know exist (like `billing`) and as a fallback for anything else resort to `ucfirst( $item['type'] )`.